src/components/HURUmap/ChartTooltip/index.stories.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+import ChartTooltip from "@/pesayetu/components/HURUmap/ChartTooltip";
+
+export default {
+  title: "Components/HURUmap/ChartTooltip",
+  argTypes: {
+    label: {
+      control: {
+        type: "text",
+      },
+    },
+    value: {
+      control: {
+        type: "text",
+      },
+    },
+    percentValue: {
+      control: {
+        type: "text",
+      },
+    },
+    group: {
+      control: {
+        type: "text",
+      },
+    },
+    groupColor: {
+      control: {
+        type: "text",
+      },
+    },
+  },
+};
+
+const Template = ({ ...args }) => <ChartTooltip {...args} />;
+
+export const Default = Template.bind({});
+
+Default.args = {
+  label: "15-24",
+  value: "1456000",
+  percentValue: "45%",
```

Comment:
```suggestion
  title: "15-24",
  value: "1456000",
  formattedValue: "45%",
```
I'm assuming we need to keep naming consistent with the other components.